### PR TITLE
Included WFK 2.5 BOMs and Archetypes

### PR DESCRIPTION
**Pull request by rafabene**

### CODE REVIEW

File: stacks.yaml [hunk 1]
```diff
@@ -1971,6 +2098,55 @@ availableArchetypeVersions:
...
+       essentialDependencies: ["org.jboss.bom.wfk:jboss-javaee-6.0-with-tools:pom:2.5.0-redhat-1", "org.jboss.bom.eap:jboss-javaee-6.0-with-hibernate:pom:6.2.0.GA"],
+   }
+
+ - &richfaces-archetype-simpleapp-wfk-25
+   id: richfaces-archetype-simpleapp-wfk-25
+   archetype: *richfaces-archetype-simpleapp-wfk
+   version: *wfk25ArchetypeRelease
+   repositoryURL:
+   labels: {
+       type: richfaces-kitchensink,
```
Comment (rafabene) [inline stacks.yaml:2122]: What "type" should we use for richfaces-archetype-simpleapp Archetype ?

File: stacks.yaml [hunk 2]
```diff
@@ -1971,6 +2098,55 @@ availableArchetypeVersions:
...
+   #   Spring Archetypes   #
+   #########################
+   
+ - &jboss-spring-mvc-archetype-wfk-25
+   id: jboss-spring-mvc-archetype-wfk-25
+   archetype: *jboss-spring-mvc-archetype-wfk
+   version: *wfk25ArchetypeRelease
+   repositoryURL:
+   labels: {
+       type: richfaces-kitchensink,
```
Comment (rafabene) [inline stacks.yaml:2141]: What "type" should we use for jboss-spring-mvc-archetype Archetype ?

File: stacks.yaml (hunk 1 above)
Comment (fbricon) [inline stacks.yaml:2122]: type is not required for non-JBoss Central archetypes. We can decide to add one later, if/when we need it

File: stacks.yaml (hunk 2 above)
Comment (fbricon) [inline stacks.yaml:2141]: This one should be spring-kitchensink

**Pull request merged**